newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -794,90 +835,117 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
-                  goToFirstPage,
-                }}
-                erroredEntry={
-                  apiError &&
-                  apiError.action === 'entryDeletion' &&
-                  apiError.itemId
-                    ? { entryId: apiError.itemId, message: apiError.message }
-                    : undefined
+                  <Text size="body2">
+                    <Trans>Player best entry</Trans>
+                  </Text>
+                </Tooltip>
+                <Switch
+                  color="primary"
+                  size="small"
+                  checked={displayOnlyBestEntry}
+                  onClick={() => setDisplayOnlyBestEntry(!displayOnlyBestEntry)}
+                />
+                <Divider orientation="vertical" />
+                <IconButton
+                  onClick={onFetchLeaderboardEntries}
+                  disabled={isRequestPending || isEditingName}
+                  tooltip={t`Refresh`}
+                  size="small"
+                >
+                  <Refresh />
+                </IconButton>
+                <Spacer />
+              </Line>
+              {apiError && apiError.action === 'entriesFetching' ? (
+                <CenteredError>
+                  <PlaceholderError
+                    onRetry={onFetchLeaderboardEntries}
+                    kind="error"
+                  >
+                    {apiError.message}
+                  </PlaceholderError>
+                </CenteredError>
+              ) : (
+                <LeaderboardEntriesTable
+                  entries={entries}
+                  customizationSettings={
+                    currentLeaderboard
+                      ? currentLeaderboard.customizationSettings
+                      : null
+                  }
+                  onDeleteEntry={entryId => onDeleteEntry(i18n, entryId)}
+                  isLoading={isRequestPending || isEditingName}
+                  navigation={{
+                    goToNextPage,
+                    goToPreviousPage,
+                    goToFirstPage,
+                  }}
+                  erroredEntry={
+                    apiError &&
+                    apiError.action === 'entryDeletion' &&
+                    apiError.itemId
+                      ? { entryId: apiError.itemId, message: apiError.message }
+                      : undefined
+                  }
+                />
+              )}
+            </div>
+          </ResponsiveLineStackLayout>
+          {isEditingAppearance ? (
+            <LeaderboardAppearanceDialog
+              open
+              leaderboardCustomizationSettings={
+                currentLeaderboard
+                  ? currentLeaderboard.customizationSettings
+                  : undefined
+              }
+              onClose={() => setIsEditingAppearance(false)}
+              onSave={async (
+                customizationSettings: LeaderboardCustomizationSettings
+              ) => {
+                try {
```

Comment:
should we catch and show an error box if it errors?

Reply:
That's what `onUpdateLeaderboard` does

Reply:
so you don't need the try here as it will not throw?

Reply:
I'm using this for the `finally` so that, whatever happens (unexpected error), the dialog is closed and you can see the admin dialog where api errors are displayed